cli/src/command/host.rs

```diff
@@ -52,10 +52,10 @@ fn setup_crate(host_path: PathBuf) -> anyhow::Result<()> {
             "# Generated by cargo-nexus, do not remove!\n",
             "#\n",
             "# This profile is used for generating proofs, as Nexus VM support for compiler optimizations is still under development.\n",
-            "[profile.release-unoptimized]\n",
-            "inherits = \"release\"\n\n",
-            "[profile.release-unoptimized.package.guest]\n",
-            "opt-level = 0\n\n",
+            "[profile.release]\n",
```

Comment:
This is unnecessary, release uses that opt-level by default. I'd remove from here and below.

Reply:
The same comment applies everywhere else.

Reply:
fixed in 923b498

Reply:
Sorry if I wasn't clear. There should be nothing about `profile.release` whatsoever. We should leave it undefined, so that the defaults are used.

Reply:
Do you mean to delete the all `[profile.release]` section? 
From 

```toml
[profile.release]
codegen-units = 1
lto = true
opt-level = 3

[profile.bench]
inherits = "release"
debug = true
```

to 
```toml
[profile.bench]
inherits = "release"
debug = true
```

Reply:
updated in 7cd20ba  
The `opt-level` is deleted, the defaults are used.